load/employerStaticFiles.js

```diff
@@ -0,0 +1,52 @@
+import http from 'k6/http';
+import { fail } from 'k6';
+import { Counter } from 'k6/metrics';
+import { parseHTML } from 'k6/html';
+
+const durationSeconds = __ENV.DURATION;
+const rate = __ENV.RATE;
+
+let successCounter = new Counter('SUCCESS_REQS');
+let failCounter = new Counter('FAIL_REQS');
+
+export let options = {
+  insecureSkipTLSVerify: true,
+  scenarios: {
+    all: {
+      executor: 'constant-arrival-rate',
+      rate,
+      timeUnit: '1s',
+      duration: `${durationSeconds}s`,
+      preAllocatedVUs: 0,
+      maxVUs: rate,
+    },
+  },
+};
+
+export default function () {
+  const indexRes = http.get(`https://hcapemployers.${__ENV.OS_NAMESPACE_SUFFIX}.freshworks.club/`);
+
+  if (indexRes.status === 200) {
+    successCounter.add(1);
```

Comment:
we should add only one success counter for this entire function, since HTML + JS counts as one load.

Reply:
Very good point, thanks!